### DISCUSSION TITLE
Add Lethal Company Mods (Please)

### DISCUSSION THREAD

**Discussion by SilvertongueRED** [ebkr/r2modmanPlus#1106] [Ideas]
Just like the title says, please add Lethal Company mods to R2ModManager as they're currently only in the Thunderstore which is windows only and I'm on a steam deck (hence why I use this great mod manager instead!). Please add these mods soon as I want them to play with my friends :)

**Comment by EllieJaybee** [comment]
It'll be released tomorrow or the day after, be patient
[Sauce](https://ptb.discord.com/channels/727304496522461185/727304496522461189/1176780292762107915)
![image](https://github.com/ebkr/r2modmanPlus/assets/47494925/6e073ee2-3d35-4739-b166-0604c8820d62)

**Reply by SilvertongueRED** [reply]
Oh yep so sorry seeing that! Appreciate the heads up again my bad.

**Comment by SilvertongueRED** [comment]
Huh weird lethal didn’t show up for me on first load into R2 mod manager buuut once i quit it and loaded it up again it did!Roland DuBoisiCloudOn Nov 26, 2023, at 4:34 PM, Crooozy ***@***.***> wrote:﻿
not there D:

**Comment by hnnhmrry** [comment]
plz i dont know how to use this platform at all, i just wanna play lethal company on mac. i dont even know how to post a discussion thing to like the general public of this website 
plz help it would be much appreciated
